docs/network/operators.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Network operators
+---
+
+Etherlink relies on operators who run nodes.
+For information on the roles of these different nodes, see [Etherlink architecture](/network/architecture).
+
+## Nodes
+
+Anyone can run Etherlink [EVM nodes](/network/evm-nodes) and [Smart Rollup nodes](/network/smart-rollup-nodes), and due to the optimistic nature of Tezos [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) it takes only one honest Smart Rollup node operator to keep Etherlink secure.
+Honest Smart Rollup nodes can catch any misbehavior by other nodes by refuting their incorrect commitments.
+
+These organizations currently run Etherlink nodes:
+
+- [The Tezos Foundation](https://tezos.foundation/)
+- [MIDL.dev](http://midl.dev/)
+- [Zeeve](https://www.zeeve.io)
+
```

Comment:
Mention https://tzkt.io/sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf/bondholders to let users find out themselves the addresses used by these organizations.

Reply:
Thanks, added this:

> You can look up the current node operators by checking the accounts that currently have bond set so they can post commitments for the Etherlink Smart Rollup, such as on the TzKT block explorer: https://tzkt.io/sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf/bondholders.